packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -50,6 +50,7 @@ export const createSeatManager = (
    * @returns {Allocation}
    */
   const getCurrentAllocation = zcfSeat => {
+    // TODO update docs that getCurrentAllocation() fails after exit
```

Comment:
We prefer that this kind of TODO include the issue number (which means creating the issue before merging the PR.)